lib/vanagon/engine/base.rb

```diff
@@ -10,7 +10,7 @@ def initialize(platform, target = nil)
         @platform = platform
         @required_attributes = ["ssh_port"]
         @target = target if target
-        @target_user = "root"
+        @target_user = @platform.target_user
```

Comment:
The pooler check creates a new pooler engine instance, which inherits from base. The test needs to be updated to expect this.